src/main/java/pro/sky/telegrambot/controller/PetController.java

```diff
@@ -22,17 +22,15 @@ public PetController(PetService petService) {
 
     @GetMapping("/{id}")
     public Pet findPet(@PathVariable int id) {
-        checkParametersForNull(id);
         return petService.findPetById(id);
     }
 
     @PostMapping("/add")
```

Comment:
```suggestion
    @PostMapping
```
POST уже говорит о том, что это добавление/создание

Reply:
Это вот прям общепринятое правило так писать или все-таки вкусовщина? Не знаю почему, но мне хочется все равно дописывать дополнительно в строке). Как-будто бы для меня так нагляднее

Reply:
Это правила REST. Сейчас твой запрос звучит как "добавь добавь" - т.е. масло масляное

Reply:
поэтому глаголы в путях писать не нужно, глаголы - это методы HTTP, они говорят о действии